content/en/docs/Reference/veneers.md

```diff
@@ -0,0 +1,446 @@
+---
+title: "Veneers"
+linkTitle: "Veneers (alpha)"
+weight: 5
+date: 2022-06-30
+---
+
+>Note: `veneers` are **ALPHA** functionality and may adopt breaking changes
+
+
+## Concept and Design
+
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions. FBCs give operator authors a [declarative and deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/). However, FBCs can get complex, especially as the number of releases and dependencies scale. We introduce the concept of a `veneer` as an approach to simplifying interacting with FBCs.
+
+In this context, there are two components to every `veneer`:
+1. An arbitrary API
+2. An executable which processes #1 and produces a valid FBC.
```

Comment:
I would prefer something like: 

“veneer is API, you can extend and create your own veeners. Here we will show the veeners supported by OPM and how they can help you out abstract the complex to generate your Operator package metadata in the FBC format. Note: the veener syntax can only help you generate FBC manifests”  

wdyt?